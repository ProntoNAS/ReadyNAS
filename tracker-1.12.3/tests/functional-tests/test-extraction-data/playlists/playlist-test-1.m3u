[playlist]
File1=http://www.apnaradio.com/live/ApnaRadio.mp3
File2=http://live.apnaradio.com:6464
File3=http://live.apnaradio.com:2424
File4=http://www.apnaradio.com/live/MaintenanceE.mp3
File5=http://www.apnaradio.com/live/MaintenanceP.mp3
NumberOfEntries=5

