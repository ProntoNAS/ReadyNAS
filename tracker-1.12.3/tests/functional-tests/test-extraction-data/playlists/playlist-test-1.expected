[TestFile]
Filename=playlist-test-1.m3u
Bugzilla=
Comment=Regular m3u playlist file

[Metadata]
a=nmm:Playlist
nfo_entryCounter=5
nfo_hasMediaFileListEntry_entryUrl=http://www.apnaradio.com/live/ApnaRadio.mp3
nfo_hasMediaFileListEntry_entryUrl=http://live.apnaradio.com:6464
nfo_hasMediaFileListEntry_entryUrl=http://live.apnaradio.com:2424
nfo_hasMediaFileListEntry_entryUrl=http://www.apnaradio.com/live/MaintenanceE.mp3
nfo_hasMediaFileListEntry_entryUrl=http://www.apnaradio.com/live/MaintenanceP.mp3
